webpack.config.js

```diff
@@ -7,7 +7,7 @@ const nodeEnv = process.env.NODE_ENV || 'development';
 const isProd = nodeEnv === 'production';
 
 module.exports = {
-  devtool: isProd ? 'hidden-source-map' : 'cheap-eval-source-map',
+  devtool: isProd ? 'hidden-source-map' : 'inline-source-map',
```

Comment:
Maybe better to use `cheap-module-source-map` regarding to [issue in webpack](https://github.com/webpack/webpack/issues/2145#issuecomment-294361203) that you linked in [this Electron issue](https://github.com/electron/electron/issues/8854)